Easydict/objc/Service/Google/EZGoogleTranslate.m

```diff
@@ -139,7 +139,8 @@ - (nullable NSString *)wordLink:(EZQueryModel *)queryModel {
                                         EZLanguageKorean, @"ko",
                                         EZLanguageFrench, @"fr",
                                         EZLanguageSpanish, @"es",
-                                        EZLanguagePortuguese, @"pt",
+                                        EZLanguagePortuguese, @"pt-PT",
+                                        EZLanguageBrazilianPortuguese, @"pt",
```

Comment:
Note that, `pt-PT` is for [Portuguese](https://translate.google.com/?sl=en&tl=pt-PT&op=translate), `pt` is for [Brazilian Portuguese](https://translate.google.com/?sl=en&tl=pt&op=translate).

```
 EZLanguagePortuguese, @"pt-PT",
EZLanguageBrazilianPortuguese, @"pt",
`